addons/message_paid_research/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "api_version": "0.1",
+  "id": "message_paid_research",
+  "name": "Participate in paid research to help make VPN better",
+  "type": "message",
+  "translatable": false,
+  "conditions": {
+    "translation_threshold": 0,
+    "locales": ["en"],
+    "javascript": "currencyCheck.js"
+  },
+  "javascript": {
+    "enable": "setDate.js"
+  },
+  "message": {
+    "id": "message_paid_research",
+    "title": "Participate in paid research to help make VPN better",
+    "subtitle": "This study will take place in April 2023, and you could receive up to $200 USD (or the equivalent in your local currency) as our thanks for your time and feedback.",
+    "badge": "survey",
+    "blocks": [
+      { "id": "c_1",
+        "type": "text",
+        "content": "We’re exploring ways to enhance the Mozilla VPN experience and want to hear your thoughts."
+      },
+      { "id": "c_2",
+        "type": "text",
+        "content": "This research can be completed right from your home computer."
+      },
```

Comment:
Looking at the mockup, it seems these two blocks should be combined into one. Is that correct?